activesupport/test/inflector_test.rb

```diff
@@ -31,12 +31,18 @@ def test_pluralize_empty_string
     assert_equal "", ActiveSupport::Inflector.pluralize("")
   end
 
-  def test_pluralize_for_words_with_non_ascii_characters
-    ActiveSupport::Inflector.inflections do |inflect|
-      inflect.uncountable "猫"
+  %w{HTTP 猫}.each do |word|
```

Comment:
Can we keep them as two tests? I don't see the problem with duplication in tests, in fact I find it easier to run only one test when needed?